### DISCUSSION TITLE
reclsmation

### DISCUSSION THREAD

**Discussion by lcyddmc** [MaaAssistantArknights/MaaAssistantArknights#10619] [Q&A]
生息演算点数没刷满就停了

**Comment by Alan-Charred** [comment]
> 生息演算点数没刷满就停了

您好，您使用的是哪个版本，卡在什么画面了？请 尝试最新版的 MAA，若还遇到同样的问题，请带卡住位置的截图和相对应的日志文件发 issue，我们会尽快优化。

**Comment by Daydreamer114** [comment]
遇到问题请前往主仓库提交 issue
